user/views.py

```diff
@@ -18,7 +18,7 @@ class CreateTokenView(ObtainAuthToken):
 
 class ManageUserView(generics.RetrieveUpdateAPIView):
     serializer_class = UserSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
The `authentication_classes` line is commented out. If you intend to use token authentication for the `ManageUserView`, you should uncomment this line. Otherwise, ensure that the intended authentication method is correctly configured.